### DISCUSSION TITLE
SD 1.5 is completely Gone its Replaced By SD 1.6 what does this mean

### DISCUSSION THREAD

**Discussion by LadyFlames** [LykosAI/StabilityMatrix#619] [General]
![SD ](https://github.com/LykosAI/StabilityMatrix/assets/137729901/0ec56cbb-9dae-4252-9f6f-4086162262d9)

**Comment by mohnjiles** [comment]
https://platform.stability.ai/docs/release-notes#stable-image-v1-release

Looks like a more cost-effective version of 1.5

**Reply by LadyFlames** [reply]
its possible
